src/Handler/Output.hs

```diff
@@ -49,10 +50,20 @@ outputStream token start = do
 
     if not $ stale now command
         then outputStream token $ start + genericLength outputs
-        else sendClose ("command no longer running" :: Text)
+        else do
+            lift $ archiveOutput command
+            sendClose ("command no longer running" :: Text)
```

Comment:
`sendClose` is type-class-polymorphic, so with `OverloadedStrings` on, Haskell doesn't know which type I mean with the literal.